kqueen_ui/blueprints/ui/views.py

```diff
@@ -749,6 +749,19 @@ class ClusterDetail(KQueenView):
     validation_hint = 'uuid'
 
     def handle(self, cluster_id):
+        def _update_status_with_floatingips(status, cluster):
+            if 'resources' not in cluster['metadata']:
+                return status
+            instances = (*cluster['metadata']['resources']['masters'], *cluster['metadata']['resources']['slaves'])
+            ip_map = {instance['ip']: instance['fip'] for instance in instances if instance.get('fip')}
+
+            for n in status['nodes']:
+                internal_ip = [ip[len('InternalIP: '):] for ip in n['ip'] if 'InternalIP' in ip][0]
+                fip = ip_map.get(internal_ip)
+                if fip:
+                    n['ip'].append('FloatingIP: {}'.format(fip))
```

Comment:
does it work on all  OS/GKE/AKS ?

Reply:
line 754, it returns status as it is

Reply:
we should provide common structure for all supported provisioners